prefs/prefs.go

```diff
@@ -58,6 +58,7 @@ type Defaults struct {
 	Project      string `ini:"project,omitempty"`
 	Environment  string `ini:"environment,omitempty"`
 	Service      string `ini:"service,omitempty"`
+	Vim          bool   `ini:"vim,omitempty"`
```

Comment:
The `vim` option would fit better into `core`. 
`defaults` Is reserved for code/organization structure.